stub/src/test/java/io/grpc/stub/ServerCallsTest.java

```diff
@@ -374,15 +438,19 @@ public Integer parse(InputStream stream) {
   }
 
   private static class ServerCallRecorder extends ServerCall<Integer, Integer> {
-    private List<Integer> requestCalls = new ArrayList<Integer>();
+    private final MethodDescriptor<Integer, Integer> methodDescriptor;
```

Comment:
This field is used in ServerCalls.java:
```
Preconditions.checkArgument(
            call.getMethodDescriptor().getType().clientSendsOneMessage(),
```

This field was previously a "private" and was always null, I turned it into a "private final" and moved it to the rest of the "private final"s.